### PR TITLE
[updater] fix bootstrap

### PR DESCRIPTION
**Pull request by raphaelgavache**
### What does this PR do?

- remove bootstrap dependency on datadog.yaml: keep the pattern of working package installation when the config file is not set
- pin agent versions that won't get deleted for default install

### CODE REVIEW

File: pkg/updater/defaults/catalog.json [hunk 1]
```diff
@@ -1,44 +1,24 @@
...
-        }
-    ]
+  "packages": [
+    {
+      "name": "datadog-agent",
+      "package": "datadog-agent",
+      "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28815219-1",
+      "platform": "linux",
+      "arch": "amd64",
+      "url": "https://dd-agent-macostesting.s3.amazonaws.com/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28815219-1_oci_amd64.tar",
```
Comment (arbll) [inline pkg/updater/defaults/catalog.json:9]: Is this the expected version you want to ship ? That bucket looks weird but maybe it was just an easy place to store temp artifacts at ?

**Review by arbll — APPROVED** [review]

File: pkg/updater/defaults/catalog.json (hunk 1 above)
Comment (chouquette) [reply pkg/updater/defaults/catalog.json:9]: We're currently sharing the same bucket for OCI and macOS among other things, so it's scary but expected.

Ultimately this is likely to be revamped as `install.datadog.com` or similar

**Review by chouquette — APPROVED** [review]
LGTM for agent platform owned file

**Pull request merged**